tesseract/tesseract_support/urdf/abb_irb2400.urdf

```diff
@@ -30,15 +30,15 @@
       <geometry>
         <mesh filename="package://tesseract_support/urdf/abb_irb2400/irb2400/visual/base_link.dae"/>
       </geometry>
-      <material name="">
+      <material name="base_link_material">
```

Comment:
@Levi-Armstrong Does this change suggest that urdf_dom had a default name, but this one doesn't?

Reply:
I will test it with urdfdom and see if I get the same error I was getting from tesseract_urdf. It looks like tesseract_motion_planners tests are only failing on kinetic. Could you run the tests on your machine and see why they are failing?

Reply:
Anonymous `material` definitions are definitely supported by `urdfdom`, so the name should not be necessary -- if you want to maintain some level of compatibility with `urdfdom` that is.

Reply:
@gavanderhoorn Thank you. I have added support for anonymous material names.